src/status_im/ui/screens/bottom_sheets/views.cljs

```diff
@@ -40,4 +40,4 @@
           (merge key-storage/migrate-account-password))]
     [quo/bottom-sheet opts
      (when content
-       [content])]))
+       [content (when options options)])]))
```

Comment:
This is the same as  `[content options]` I believe, as `(when false true)` would return nil (the only difference I can think is of is if you expect `options` to be literally `false`)